client/keys/import_test.go

```diff
@@ -25,7 +25,7 @@ func Test_runImportCmd(t *testing.T) {
 	kbHome := t.TempDir()
 	kb, err := keyring.New(sdk.KeyringServiceName(), keyring.BackendTest, kbHome, mockIn)
 
-	clientCtx := client.Context{}.WithKeyring(kb)
+	clientCtx := client.Context{}.WithKeyring(kb).WithKeyringDir(kbHome)
```

Comment:
This test was failing, unrelated, but i wanted all tests to be passing so included in this PR.